publications/joss/paper.md

```diff
@@ -74,17 +75,17 @@ telescopes, such as the James Webb Space Telescope [e.g., @JWST2023PASP] and CRI
 from extrasolar planets [e.g., @Alderson2023Nature], and brown dwarfs [e.g., @Miles2023ApJ] atmospheres. Without an accurate interpretation of this data, the main objectives 
 of space missions will not be fully accomplished. Different analytical and statistical methods, such as the chi-squared-test, 
 Bayesian statistics as well as radiative-transfer atmospheric modeling packages have been developed 
-[e.g., @batalha2019picaso; @MacDonald2023POSEIDON] to interpret the spectra. They utilize either forward- and/or retrieval-radiative transfer modeling to analyze the spectra and 
+[e.g., @batalha2019picaso; @MacDonald2023] to interpret the spectra. They utilize either forward- and/or retrieval-radiative transfer modeling to analyze the spectra and 
 extract physical information, such as atmospheric temperature, metallicity, carbon-to-oxygen ratio, and surface gravity 
 [@line2014systematic; @Iyer2023Sphinx; @Marley2015]. These atmospheric models rely on generating the physics and chemistry of these atmospheres for a wide range of thermal structures 
 and compositions. In addition to Bayesian-based techniques, machine learning and deep learning methods have been developed in recent years 
 for various astronomical problems, including confirming the classification of light curves for 
 exoplanet validation [e.g., @Valizadegan2021], recognizing molecular features  [@Zingales2018ExoGAN] as well as interpreting brown dwarfs spectra using Random Forest technique 
-[e.g., @Lueber2023RandomForesr_BDs]. Here, we present one of the first applications of deep learning and convolutional neural networks on the interpretation brown dwarf 
+[e.g., @Lueber2023RandomForesr_BDs]. Here, we present one of the first applications of deep learning and convolutional neural networks on the interpretation of brown dwarf 
 atmospheric datasets. The configuration of a CNN and the key concepts can be found in [@Goodfellow_2016DeepLearning; @KIRANYAZ2021].
 
 With the continuous observation of these objects and the increasing amount of data, there is a 
-critical need for a systematic pipeline to quickly explore the datasets and extract important physical from them. In the future we can expand our pipeline to exoplanet atmospheres, and use it to provide insights about the diversity of exoplanets and brown dwarfs' 
+critical need for a systematic pipeline to quickly explore the datasets and extract important physical quantities from them. In the future we can expand our pipeline to exoplanet atmospheres, and use it to provide insights about the diversity of exoplanets and brown dwarfs' 
```

Comment:
I added "quantities" to make this sentence grammatically correct, but please feel free to edit to something else if it's more appropriate.

Reply:
Great catch. Thanks! I changed that to "parameters".